cuckoo/query.py

```diff
@@ -172,7 +173,8 @@ def many(
 
         return self._many_finalizer(
             node=inner_query,
-            returning_fn=inner_query._build_many_models,
+            streaming_fn=inner_query._build_many_models_stream,  # for `yielding` and `returning`
```

Comment:
part 0 of the feature: only, and only for the `Query().many().yielding()` query do we add this feature. It is trivial to add the same to `Update().many().yielding()`, although I don't know if we ever use this as we usually do `affected_rows()` instead. For other queries like `Query().aggregate().yield_with_nodes()` the feature would make sense, but it would be a breaking change as the response would no longer be 2 generators (`tuple[Generator[int], Generator[Model]]`) as such response would require knowledge of the _entire_ response upfront. instead, the response would change to a generator that yields a `key` and `value`, where the `key` can be either `"aggregate"` or `"node"` - I guess I am confusing you guys more...

I would say, let's test how this goes in production and think of other queries if we see need.

Reply:
Agreed